style.css

```diff
@@ -0,0 +1,1408 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(./assets/fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Crete Round";
+  src: url(./assets/fonts/CreteRound-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Poppins";
+  src: url(./assets/fonts/Poppins-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Inter";
+  src: url(./assets/fonts/Inter-VariableFont_slntwght.ttf);
+}
+
+@font-face {
+  font-family: "InaiMathi";
+  src: url(./assets/fonts/Murasuit.ttf);
+}
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* button interaction */
+
+.button:visited {
+  color: white;
+}
+
+.button:hover {
+  text-decoration: underline;
+  color: aqua;
+}
+
+.button:active {
+  color: yellow;
+}
+
+/* main code */
+
+.desktop-header {
+  display: none;
+}
+
+.toolbar {
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+  align-items: center;
+  background: #3c3a39;
+  padding: 5%;
+  min-height: 5vh;
+}
+
+/* menu strt */
+.m-menu {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background: #3c3a39;
+  mix-blend-mode: normal;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  overflow-x: hidden;
+  transition: 0.5s;
+}
+
+.m-menu-content {
+  position: relative;
+  top: 100px;
+  width: 80%;
+  padding-left: 40px;
+}
+
+.m-menu-content a {
+  border-bottom: 1px solid #6f6c6b;
+}
+
+.m-menu a {
+  padding: 8px;
+  text-decoration: none;
+  font-size: 36px;
+  color: #fff5e1;
+  display: block;
+  transition: 0.3s;
+}
+
+.m-menu a:hover,
+.m-menu a:focus {
+  color: #f1f1f1;
+}
+
+.m-menu .closebtn {
+  position: absolute;
+  top: 20px;
+  right: 45px;
+  font-size: 60px;
+}
+
+/* menu end */
+
+.humberger {
+  display: flex;
+  flex: 1;
+  padding-right: 12px;
+}
+
+.humberger:hover {
+  color: white;
+}
```

Comment:
- [ ] Please show attention to your class names and your CSS selectors. There aren't any elements with this name in your HTML.